remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala

```diff
@@ -184,8 +182,6 @@ class NettyTransportSettings(config: Config) {
     case value => value.toInt
   }
 
-  val SslSettings: Option[SSLSettings] = if (EnableSsl) Some(new SSLSettings(config.getConfig("security"))) else None
```

Comment:
strictly speaking: we can't remove a public field without deprecating it first

Reply:
let's keep it then.